Fix version in FAQ (#2101)
woodpecker-ci/woodpecker: docs/src/pages/faq.md

Woodpecker is having two different kinds of releases: **stable** and **next**.

> The **stable** releases (currently version 1.0) is a long-term supported (LTS) stable version. The stable releases are only getting bugfixes.

Reviewer: ```suggestion
The **stable** releases (currently version 1.0) are long-term supported (LTS) stable versions. The stable releases are only getting bugfixes.
```
